Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -39,6 +39,7 @@ public async Task InitDefaultCustomCommands()
             {
                 customCommands.AddRange(new List<AddCustomCommandsCommand>
                 {
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.UselessFeatures.BotCommands.GoogleCommand", @"-google\s*(?<Search>.*)", serverId),
```

Comment:
nie wiem jak to ty z @Marcin99b myśleliście nad tym, ale może lepiej by było zrobić "-google -search CośTam". Trochę dłuższe, ale jakieś bardziej czytelne, ale w sumie sam nie wiem.

Reply:
ta opcja jest tą domyślną komendą, ale skrót w postaci -google tekst uważam, że jest łatwiejszy w użyciu

Reply:
czyli, że można używać obie wersje?

Reply:
tak, z czego te custom commands oczywiście tylko na tych co dodano
ale niezależnie czy jest dla danego serwera custom command, zawsze działają również te domyślne

Reply:
o kurcze... kiedy przerabiałem mechanizm IBotCommand to nie wiedziałem, że tak może być, więc teraz na szybko sprawdziłem i wyszło mi, że czasami jest tak jak ty mówisz załączy jako domyślna a czasem mimo to załączy jako CustomCommand (mimo, że jest w tej domyślnej postaci) i czasem dobrze mimo to obsłuży a czasem nie. Więc muszę chyba tam trochę inaczej to wymyśleć i też jednocześnie spojrzyć na to inaczej.